src/docfx/Errors.cs

```diff
@@ -373,16 +373,16 @@ public static Error ViolateSchema(SourceInfo source, string message)
         /// <summary>
         /// The input value type does not match expected value type.
         /// </summary>
-        /// Behavior: ✔️ Message: ❌
+        /// Behavior: ✔️ Message: ✔️
         public static Error UnexpectedType(SourceInfo source, object expectedType, object actualType)
-            => new Error(ErrorLevel.Warning, "unexpected-type", $"Expect type '{expectedType}' but got '{actualType}'", source);
+            => new Error(ErrorLevel.Warning, "unexpected-type", $"Expected type '{expectedType}' but got '{actualType}'", source);
 
         /// <summary>
         /// The input value is not defined in a valid value list.
         /// </summary>
-        /// Behavior: ✔️ Message: ❌
-        public static Error UndefinedValue(SourceInfo source, object value, IEnumerable<object> validValues)
-            => new Error(ErrorLevel.Warning, "undefined-value", $"Value '{value}' is not accepted. Valid values: {Join(validValues)}", source);
+        /// Behavior: ✔️ Message: ✔️
+        public static Error InvalidValue(SourceInfo source, string name, object value)
+            => new Error(ErrorLevel.Suggestion, "invalid-value", $"Invalid value for '{name}': '{value}'", source);
```

Comment:
The ErrorLevel should be defined in schema instead of hardcoding as Suggestion.

Reply:
This will be done in docs build